compiler/plc_ast/src/ast.rs

```diff
@@ -688,7 +652,7 @@ impl DataType {
             DataType::VarArgs { referenced_type, .. } => referenced_type
                 .as_ref()
                 .and_then(|it| DataTypeDeclaration::get_name(it.as_ref()))
-                .or(Some(VOID_TYPE)),
+                .or(Some("VOID")), // TODO: Error prone if typesystem::VOID_TYPE ever changes
```

Comment:
Is there a reason for not using the VOID_TYPE constant here?

Reply:
IIRC because of the typesystem dependency; I wanted to make the AST dependency free from all other workspaces since that kinda make sense but open for ideas for such cases

Reply:
I think it is legit to argue, that why did the AST decide here to sneak in a VOID in the case of a VarArgs that cannot find it's referenced type (which feels like a broken situation tbh). While the API anyway returns an Option, so it would be OK to return None and let the caller decide to fall back to Void if required.

Reply:
Nice solution, resolved with commit https://github.com/PLC-lang/rusty/pull/889/commits/6ee0e8bb0c9d6d85a51a9b236b79ab64c88c444e